mlflow/projects/databricks.py

```diff
@@ -40,8 +40,8 @@ def before_run_validations(tracking_uri, backend_config):
     if backend_config is None:
         raise ExecutionException("Backend spec must be provided when launching MLflow project "
                                  "runs on Databricks.")
-    if not tracking.utils._is_databricks_uri(tracking_uri) and \
-            not tracking.utils._is_http_uri(tracking_uri):
+    if not utils.uri._is_databricks_uri(tracking_uri) and \
```

Comment:
some of these functions are used outside the direct package. Does it make sense for use to make them public?